.github/workflows/deploy-dev.yml

```diff
@@ -153,20 +155,22 @@ jobs:
                 fi
             done
           done
-      
-      - name: Deploy to OpenShift Dev
+
+      - name: Set Route Host
         run: |
-          # Create a valid helm release name and route host (lowercase)
           ROUTE_HOST="${{ steps.app_name.outputs.name }}-dev.${{ secrets.APPS_DOMAIN }}"
-          
+          echo "ROUTE_HOST=${ROUTE_HOST}" >> $GITHUB_ENV
```

Comment:
Pls output to GITHUB_OUTPUT, for now it's a only used in one other process for one other use case. See the usage of GITHUB_ENV: 
> Use an environment variable if you want the value to be available to all processes in following steps of the same job. Environment variables set via GITHUB_ENV don't carry over to other jobs.

Reply:
Kan met GITHUB_OUTPUT ook niet omdat de ROUTE_HOST als secret wordt gedetect, dus maakt dan niet uit

Reply:
Hm? Wat detecteert ROUTE_HOST als een secret? Het zou geen secret moeten zijn

Reply:
> Use an environment variable if you want the value to be available to all processes in following steps of the same job. Environment variables set via GITHUB_ENV don't carry over to other jobs.

In de documentatie staat trouwens dit: "You can make an environment variable available to any subsequent steps in a workflow job by defining or updating the environment variable and writing this to the GITHUB_ENV environment file."
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Ja, dus in dezelfde job, de deploy en test zijn twee verschillende jobs